### ISSUE TITLE
List of accepted encodings not accurate for Set-Content

### ISSUE THREAD

**Issue by RandomNoun7** [MicrosoftDocs/PowerShell-Docs#4192]
The list of acceptable values to pass to the `-Encoding` parameter is not correct. The list as as follows:
```
ascii, bigendianunicode, oem, unicode, utf7, utf8, utf8BOM, utf8NoBOM, utf32
```

As of the version:

```
Name                           Value
----                           -----
PSVersion                      6.2.0
PSEdition                      Core
GitCommitId                    6.2.0
OS                             Linux 4.17.0-9-generic #10-Ubuntu SMP Wed Aug 22 13:33:37 UTC 2018
Platform                       Unix
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

I have confirmed that this list as also the same for the following version:
```
Name                           Value
----                           -----
PSVersion                      6.1.1
PSEdition                      Core
GitCommitId                    6.1.1
OS                             Microsoft Windows 10.0.14393
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```
This issue is related to #4191

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: b874c458-21c1-4127-cbce-515e81a9d779
* Version Independent ID: e4684775-c0df-ad9a-7eef-135db60ada1b
* Content: [Set-Content (Microsoft.PowerShell.Management)](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.management/set-content?view=powershell-6)
* Content Source: [reference/6/Microsoft.PowerShell.Management/Set-Content.md](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Management/Set-Content.md)
* Product: **powershell**
* Technology: **powershell-cmdlets**
* GitHub Login: @daxian-dbw
* Microsoft Alias: **dongbow**

**Comment by sdwheeler** [comment]
See also related #4155

**Comment by RandomNoun7** [comment]
This issue is not properly addressed by #4155 or any of the associated pull requests.

**Comment by DCtheGeek** [comment]
@sdwheeler and I realized that while we were doing 3-5.1 today. We'll close it with a new PR. Thanks for pointing this out.
